src/components/common/video-player/VideoPlayer.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC, useEffect, useRef } from 'react';
+import Hls from 'hls.js';
+import { useDispatch } from 'react-redux';
+interface VideoPlayerProps {
+  src: string;
+  title: string;
+  poster: string;
+}
+
+const VideoPlayer: FC<VideoPlayerProps> = ({ src, poster, title }) => {
+  const videoRef = useRef<HTMLVideoElement>(null);
+  const dispatch = useDispatch();
+  const hls = new Hls();
```

Comment:
I would move this initialization inside `useEffect` especially if you're gonna use SSR in the future

Reply:
fixed